RELEASING.md

```diff
@@ -4,8 +4,8 @@ Currently the only external artefact for a release is the data generator image i
 
 The automation builds and pushes images to quay on push to:
 1. main (image will be tagged as main eg quay.io/streamshub/flink-examples-data-generator:main )
-2. branches named releases/** (image will be tagged as branch name eg quay.io/streamshub/flink-examples-data-generator:releases/0.0)
-3. tag push (image will be tagged with the git tag, for example git tag v0.0.1 -> quay.io/streamshub/flink-examples-data-generator:releases/v0.0.1)
+2. branches named release-** (image will be tagged as branch name, for example branch release-0.0 -> quay.io/streamshub/flink-examples-data-generator:release-0.0)
```

Comment:
`release-0.0` looks really strange to me. I don't want to say it is bad, it is just my feeling. I wonder if having `release-0.0.x` wouldn't be better 🤔 Guess we can talk about it later if anyone else will have same feelings.

Reply:
Agreed, release-0.0.x looks less like a release tag. Pushed up change.